cmd/gardener-extension-provider-local/app/app.go

```diff
@@ -262,7 +278,9 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 		},
 	}
 
-	aggOption.AddFlags(cmd.Flags())
+	flags := cmd.Flags()
+	aggOption.AddFlags(flags)
+	flags.StringVar(&opts.HealthProbeBindAddress, "health-bind-address", ":8081", "bind address for the health server")
```

Comment:
Shouldn't this flag go to the manager options?

Reply:
Yes but then changes need to be made [here](https://github.com/gardener/gardener/blob/cf1439242213e86fe400c83c10e05943d759d72a/extensions/pkg/controller/cmd/options.go#L185). And everything that usages extension/pkg/cmd will have that flag. If we are good with it, I think here https://github.com/gardener/gardener/pull/5380#issuecomment-1041179214 we should choose the second option, i.e. add `HealthProbeBindAddress` to manager options.

Reply:
Yeah, I think it makes sense that everybody gets this automatically after revendoring (less duplicating in `n` extension repos).

Reply:
I will update the PR accordingly.

Reply:
3435b89f64ba2610bb08f6626c6200a027565a48 I have updated the changes.